csuntechlab/iSTART
# STR-105 -- FE: Images are not loading correctly on all devices

# Description
Ensure that images in intervention group load correctly
  
# Testing
1. check to see if images load locally
2. check dev environment to see that they do
